.github/workflows/tests.yaml

```diff
@@ -0,0 +1,79 @@
+name: Tests and linting
+
+on:
+    push:
+    pull_request:
+    schedule:
+        - cron:  '0 3 * * *'
+
+jobs:
+    unit-tests:
+        runs-on: ubuntu-latest
+
+        strategy:
+            matrix:
+                php-version: ['7.3']
+                dependencies: ['']
+                include:
+                    - { php-version: '7.3', dependencies: '--prefer-lowest --prefer-stable' }
+
+        name: Unit tests - PHP ${{ matrix.dependencies }}
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Setup PHP
+              uses: shivammathur/setup-php@v2
+              with:
+                  php-version: ${{ matrix.php-version }}
+                  extensions: mbstring, intl, zip
+                  coverage: xdebug
+                  tools: composer:v2
+
+            - name: Install dependencies
+              run: composer update --no-interaction ${{ matrix.dependencies }}
+
+            - name: Run tests
+              env:
+                  COLUMNS: 120
```

Comment:
Is this needed for something? 

(I use it in Steward, because there are tests comparing console output - but it may not be the case here.)